themes/gnuradio-bootstrap/assets/css/style.scss

```diff
@@ -609,3 +609,27 @@ body {
 .gr-btn:hover{
   color: $gr-dark-gray;
 }
+
+.flex-container {
```

Comment:
doesn't that potentially have side effects on other pages as well?

Reply:
nope, flex-container isnt used anywhere else